bucket/azuredatastudio-insiders.json

```diff
@@ -1,22 +1,15 @@
 {
     "homepage": "https://docs.microsoft.com/en-us/sql/azure-data-studio",
-    "version": "1.12.2",
+    "version": "nightly",
     "description": "A cross-platform database tool for data professionals using the Microsoft family of on-premises and cloud data platforms.",
     "license": "MIT",
-    "bin": "bin\\azuredatastudio.cmd",
+    "bin": "bin\\azuredatastudio-insiders.cmd",
     "shortcuts": [
         [
-            "azuredatastudio.exe",
-            "Azure Data Studio"
+            "azuredatastudio-insiders.exe",
+            "Azure Data Studio - Insiders"
         ]
     ],
-    "url": "https://github.com/Microsoft/azuredatastudio/releases/download/1.12.2/azuredatastudio-windows-1.12.2.zip",
-    "hash": "89c0b9ef1223f6010802556524ee3b5042e7ebfa5e5513412bc35cbeac707b7f",
-    "extract_dir": "azuredatastudio-windows",
-    "checkver": {
-        "github": "https://github.com/Microsoft/azuredatastudio"
-    },
-    "autoupdate": {
-        "url": "https://github.com/Microsoft/azuredatastudio/releases/download/$version/azuredatastudio-windows-$version.zip"
-    }
+    "url": "https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-archive/insider#/dl.7z",
```

Comment:
Move `url` and `extract_dir` after `license` property

Reply:
Both `url` and `extract_dir` are already after the `license` property

Reply:
Not directly. Let me rephrase sentence to be more clear.

Put `url` and `hash` directly after `license` property and before `bin` property.